ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -913,6 +1175,51 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P4S Curtain Call Debuffs',
+      // Durations could be 12s, 22s, 32s, and 42s
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: 'AF4', capture: true }),
+      condition: (data, matches) => {
+        return (data.me === matches.target && data.act === 'curtain');
+      },
+      response: (data, matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = curtainCallOutputStrings;
+
+        data.curtainCallGroup = Math.ceil(((parseFloat(matches.duration)) - 2) / 10);
+
+        if (data.curtainCallGroup === 1)
+          return { alarmText: output.group!({ num: data.curtainCallGroup }) };
+        return { infoText: output.group!({ num: data.curtainCallGroup }) };
+      },
+    },
+    {
+      id: 'P4S Curtain Call Reminders',
+      // Alarms for the other groups
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: 'B7D', capture: true }),
+      condition: (data) => data.act === 'curtain',
+      preRun: (data) => data.curtainCallTracker = (data.curtainCallTracker ?? 0) + 1,
+      delaySeconds: (_data, matches) => parseFloat(matches.duration),
+      suppressSeconds: 1,
+      alarmText: (data, _matches, output) => {
```

Comment:
```suggestion
      infoText: (data, _matches, output) => {
```

Maybe infoText (or alertText) instead of alarmText for the reminder?